pkg/kine/drivers/sqlite/schema.go

```diff
@@ -13,10 +14,47 @@ import (
 // already
 
 const (
-	databaseSchemaMajorVersion = 1
-	databaseSchemaMinorVersion = 0
+	databaseSchemaMajorVersion int16 = 0
```

Comment:
I think we can safely inline these 2 constants in the definition of `databaseSchemaVersion` below, so that someone reading that line immediately knows the version.

Reply:
Just `databaseSchemaVersion = NewSchemaVersion(1, 0)`?

Reply:
Yes

Reply:
I realized it's in the last commit already :D